mne/decoding/transformer.py

```diff
@@ -157,92 +159,122 @@ class EpochsVectorizer(TransformerMixin):
 
     Attributes
     ----------
-    n_channels : int
+    n_channels_ : int
         The number of channels.
-    n_times : int
-        The number of time points.
+    n_epochs_ : int
+        The number of data samples.
```

Comment:
`inverse_transform` can be applied to a different dataset. e.g.
```
xd.fit(X_train, y_train)
X_test_trans = xd.transform(X_test)
xd.inverse_transform(X_test_trans)
```

So `n_epochs` depends on `X` and not on a parameter. Let me know if it's unclear.